### PR TITLE
Do not trigger new workflows for each files found by default

### PR DESCRIPTION
**Pull request by marcocast**

### CODE REVIEW

File: FileFolderManagement/resources/catalog/file_trigger.xml [hunk 1]
```diff
@@ -1,69 +1,166 @@
...
+    name="File Trigger" projectName="1. Basic Workflows"
+    priority="normal"
+    onTaskError="continueJobExecution"
+     maxNumberOfExecution="2"
>
+  <variables>
+    <variable name="FOLDER_TO_MONITOR" value="/tmp/trigger" />
+  </variables>
+  <description>
+    <![CDATA[ Process a file whenever the file is created in a specified folder. For the sake of this example the monitor of the specified folder is limited (see loop code). ]]>
```
Comment (jrochas) [inline FileFolderManagement/resources/catalog/file_trigger.xml]: Can you specify limited in what in the description? Limited in time? Limited in number of files?

File: FileFolderManagement/resources/catalog/file_trigger.xml [hunk 2]
```diff
@@ -1,69 +1,166 @@
...
-		<task name="FileTrigger">
-			<description>
-        <![CDATA[ The simplest task, ran by a groovy engine. ]]>
+  <genericInformation>
+    <info name="bucketName" value="1. Basic Workflows"/>
+    <info name="group" value="public-objects"/>
+    <info name="workflow.icon" value="/automation-dashboard/styles/patterns/img/wf-icons/file_trigger.png"/>
+  </genericInformation>
+  <taskFlow>
+    <task name="FileTrigger"
```
Comment (jrochas) [inline FileFolderManagement/resources/catalog/file_trigger.xml]: Weird formating

File: FileFolderManagement/resources/catalog/file_trigger.xml [hunk 3]
```diff
@@ -1,69 +1,166 @@
...
-					<code language="groovy">
+      <variables>
+        <variable name="FOLDER_TO_MONITOR" value="/tmp/trigger" inherited="true" />
+      </variables>
+      <genericInformation>
+        <info name="task.icon" value="/automation-dashboard/styles/patterns/img/wf-icons/file_trigger.png"/>
+      </genericInformation>
+      <selection>
+        <script
+         type="static" >
```
Comment (jrochas) [inline FileFolderManagement/resources/catalog/file_trigger.xml]: Why is it formatting the xml parameters like elements?

File: FileFolderManagement/resources/catalog/file_trigger.xml [hunk 4]
```diff
@@ -1,69 +1,166 @@
...
-    <![CDATA[ Trigger a job execution whenever a file is created in a specified folder. ]]>
+<job
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xmlns="urn:proactive:jobdescriptor:3.10"
+     xsi:schemaLocation="urn:proactive:jobdescriptor:3.10 http://www.activeeon.com/public_content/schemas/proactive/jobdescriptor/3.10/schedulerjob.xsd"
+    name="File Trigger" projectName="1. Basic Workflows"
+    priority="normal"
+    onTaskError="continueJobExecution"
+     maxNumberOfExecution="2"
>
```
Comment (Aminelouati) [inline FileFolderManagement/resources/catalog/file_trigger.xml]: May be u can remove this ^M charcter

File: FileFolderManagement/resources/catalog/file_trigger.xml [hunk 5]
```diff
@@ -1,69 +1,166 @@
...
+selected = org.ow2.proactive.scripting.helper.selection.SelectionUtils.checkHostName("slave3")
+]]>
+          </code>
+        </script>
+      </selection>
+      <scriptExecutable>
+        <script>
+          <code language="groovy">
             <![CDATA[
 inputFilesFolder = variables.get("FOLDER_TO_MONITOR");
```
Comment (mbenguig) [inline FileFolderManagement/resources/catalog/file_trigger.xml]: "Omitting the "def" keyword puts the variable in the bindings for the current script and groovy treats it (mostly) like a globally scoped variable"

**Pull request merged**